server/src/routes/user.ts

```diff
@@ -47,16 +68,16 @@ export const createUser = async (
   const { username, password } = req.body;
   try {
     const builtUser = await User.create({ username, password });
-    const token = await createToken(builtUser?.id || "", builtUser.id);
+    const token = await createToken(builtUser?.id || "", builtUser.password);
```

Comment:
now this token is different than the other token call and a create user will _NOT_ log a user in, now u gotta create and then login, where as it worked before. how would you fix this?

Reply:
```
const token = await createToken(builtUser?.id || "", builtUser.password);
```
creates a far different token than
```
    const token = await createToken(userFound.password, userFound?.id);
```
note that the options are reversed. the double id was correct, it just should have only been the one field.